.swift-format

```diff
@@ -11,7 +11,7 @@
   "lineBreakBeforeControlFlowKeywords": false,
   "lineBreakBeforeEachArgument": false,
   "lineBreakBeforeEachGenericRequirement": false,
-  "lineLength": 100,
+  "lineLength": 140,
```

Comment:
Hmm 140 might be too much, I prefer to keep it at 80-120 so it's the most accessible and it fits in my macbook's screen without too much trouble (that's just my experience though)

Reply:
It's the one in [vapor/contributing](https://github.com/vapor/contributing/blob/main/.swift-format), once the development for the next release ends I might even remove the `.swift-format` file in PassKit and just use Vapor's default one